ksql-engine/src/test/java/io/confluent/ksql/integration/EndToEndIntegrationTest.java

```diff
@@ -156,6 +163,7 @@ public void shouldSelectFromPageViewsWithSpecificColumn() throws Exception {
     }
 
     assertEquals(expectedPages, actualPages);
+    assertThat(IntegrationTestHarness.consumedCount.intValue(), greaterThan(0));
```

Comment:
Can these assert an exact number rather than just gt zero?

Reply:
This is a bit trickier than it is worth IMO, because the produced and consumed messages to internal topics would also be counted. So this would look like a magic number in a lot of tests, and I think is not as nice as just asserting that the value has changed. Besides, the point of this test is not to validate that interceptors work correctly. It is just to validate that they are plugged in correctly. The present assertion achieves that IMO.